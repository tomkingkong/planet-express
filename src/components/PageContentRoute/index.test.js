import React from 'react';
import { shallow } from 'enzyme';

import { PageContentRoute } from '.';

describe('PageContentRoute component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<PageContentRoute />);
    expect(wrapper).toMatchSnapshot();
  });
});