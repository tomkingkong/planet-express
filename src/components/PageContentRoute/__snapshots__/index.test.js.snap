// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContentRoute component should match snapshot 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/launch"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/profile"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/news"
  />
  <Route
    component={[Function]}
    path="/ISS/*"
  />
  <Route
    component={[Function]}
    path="/journey"
  />
</Switch>
`;
